shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -48,6 +48,15 @@
 // The parent view handles clipping to its subviews.
 @interface ChildClippingView : UIView
 
+// Adds a blur filter to its layers. Returns NO if Apple's API has changed and
+// blurred backdrop filters cannot be applied, otherwise returns YES.
+- (BOOL)applyBlurBackdropFilters:(NSArray*)blurRadii;
+
+// The UIView used to extract the gaussianBlur filter. This must be a UIVisualEffectView
+// initalized with UIBlurEffect to extract the correct filter. Made a public property
+// for custom unit tests.
+@property(nonatomic, strong) UIView* viewToExtractFrom;
```

Comment:
nits:
Use retain instead of strong to indicate it is MRC.

Also, is there a better name we can use for this property? The comment is descriptive but a better name would make this property easier to understand. 
Maybe just `blurEffectView` or `viewToExtractBlurFilter`